include/ocpp/v201/messages/Authorize.hpp

```diff
@@ -19,7 +19,7 @@ namespace v201 {
 struct AuthorizeRequest : public ocpp::Message {
     IdToken idToken;
     std::optional<CustomData> customData;
-    std::optional<CiString<5500>> certificate;
+    std::optional<CiString<10000>> certificate;
```

Comment:
This could probably be reported via `FieldLength` variable for the `AuthorizeRequest.certificate` instance which should make this change backwards-compatible with 2.0.1